webapp/components/test-search.js

```diff
@@ -174,30 +176,37 @@ const QUERY_SEMANTICS = QUERY_GRAMMAR.createSemantics().addOperation('eval', {
   _terminal: function() {
     return this.sourceString;
   },
+  Root: (r) => {
+    const ps = r.eval();
+    return ps.length === 0
+      ? emptyQuery
+      : ps.length === 1
+        ? ps[0]
+        : { and: ps };
```

Comment:
Is this changing the default interpretation of `q1 q2`? Looking in https://github.com/web-platform-tests/wpt.fyi/blob/master/api/query/README.md I can't spot if that's equivalent to `q1 and q2` or `q1 or q2`. I could experiment to find out, but it'd be very good to document that.

Reply:
Only for the case that either q1 or q2 is a different combination (e.g. `all(...)`), which was previously unsupported. The default is still exists as a root.